pcdsdevices/interface.py

```diff
@@ -1234,7 +1234,7 @@ def state(self):
             if method_name.startswith('wm_'):
                 state_name = method_name.replace('wm_', '', 1)
                 wm_state = getattr(device, method_name)
-                diff = wm_state()
+                diff = abs(wm_state())
                 if diff < closest:
```

Comment:
closest = 0.5 seems like a large margin for error, but it's not really a show stopper